### DISCUSSION TITLE
Possibilidade de habilitar Dependabot

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#36] [Ideas]
O Dependabot é uma ferramenta do GitHub que ajuda a manter as dependências do projeto atualizadas automaticamente, reduzindo a carga de trabalho manual para acompanhar e aplicar atualizações de segurança e correções de bugs. Ele analisa regularmente o projeto em busca de dependências desatualizadas e cria pull requests automaticamente para atualizá-las para as versões mais recentes compatíveis.

Links úteis:

- **Documentação oficial do Dependabot no GitHub:** Esta é a fonte oficial para entender como o Dependabot funciona e como configurá-lo em seus repositórios.
[Documentação do Dependabot](https://docs.github.com/pt/github/administering-a-repository/configuration-options-for-dependency-updates)

- **Exemplo de configuração do Dependabot:** Você pode encontrar exemplos práticos de configuração do Dependabot em repositórios públicos no GitHub.
[Exemplo de Configuração do Dependabot](https://github.com/search?q=dependabot.yml)

- **Guia prático para começar com o Dependabot**: Se você é novo no Dependabot, pode ser útil seguir um guia passo a passo para começar.
[Guia para Iniciantes no Dependabot](https://about.gitlab.com/blog/2021/11/18/github-actions-with-dependabot/)

- **GitHub Actions com Dependabot**: Integrar o Dependabot com GitHub Actions pode ser uma maneira poderosa de automatizar ainda mais o processo de atualização de dependências.
[Integração com GitHub Actions](https://github.com/features/actions)

**Comment by andrewarosario** [comment]
[Vídeo da Loiane sobre](https://www.youtube.com/watch?v=1vqJ1_AAcUg&ab_channel=LoianeGroner)

**Comment by FelipeDuarteLuna** [comment]
Bom dia @andrewarosario,

Tudo bom ?

Após Mentoria 1.0 eu fiz o trampo de habilitar o Dependabot e o Git Guardian, para efeitos de testes e como funciona. E realmente eleva o nível de segurança da aplicação e o profissionalismo.

Sou a favor de implementar não somente o Dependabot, quanto o Git guardian.

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/949abdac-e55c-42b7-8be5-9b8ffc61ca6b)
